docs-vim.js

```diff
@@ -37,12 +38,23 @@ vim.normal_keydown = function (e) {
     keyMap[vim.keys.move[2]] = "ArrowUp";
     keyMap[vim.keys.move[3]] = "ArrowRight";
 
+    if (e.key.match(/\d+/)) {
+        num += e.key.toString();
+    }
+
     if (e.key in keyMap) {
         e.key = keyMap[e.key];
     }
 
     if (e.key.indexOf("Arrow") == 0 || e.key == "Delete") {
-        docs.pressKey(docs.codeFromKey(e.key));
+	if (num != '') {
```

Comment:
Can we re-write this to:

```javascript
if (vim.num.length == 0 || isNaN(vim.num)) {
    vim.num = "1";
}
for (var i = 0; i < Number(vim.num); i++) {
    docs.pressKey(docs.codeFromKey(e.key));
}
vim.num = "";
```
To cut down on nested branches slightly + explicitly handle the case where, eg., ```num = 'a'``` (maybe in that case we should default to `vim.num = "0"` or alert an error to the user?).

Reply:
Also, please stick to double-quotes for strings for consistency :)